cli-test/__snapshots__/cli.test.js.snap

```diff
@@ -40,19 +40,18 @@ Object {
 
   Options:
 
-    -h, --help                                  output usage information
-    -V, --version                               output the version number
-    -s, --silent                                Silent nps output
-    -p, --parallel <script-name1,script-name2>  Scripts to run in parallel (comma seprated)
-    -c, --config <filepath>                     Config file to use (defaults to nearest package-scripts.yml or package-scripts.js)
-    -l, --log-level <level>                     The log level to use (error, warn, info [default])
-    -r, --require <module>                      Module to preload
+    -h, --help               output usage information
+    -V, --version            output the version number
+    -s, --silent             Silent nps output
+    -c, --config <filepath>  Config file to use (defaults to nearest package-scripts.yml or package-scripts.js)
+    -l, --log-level <level>  The log level to use (error, warn, info [default])
+    -r, --require <module>   Module to preload
 
 Available scripts (camel or kebab case accepted)
 
-test - echo \"test script\"
-lint - echo \"lint.default\"
-lint.sub.thing - this is a description - echo \"deeply nested thing\"
+[32mtest[39m - [90mecho \"test script\"[39m
```

Comment:
Actually, those are special characters for coloring the output. That's from `chalk`. The editor won't show those characters. 😀